### PR TITLE
 Migrate KS App Base to Python3, Jira:- FP-5100

### PR DESCRIPTION
**Pull request by amitasthana**

### CODE REVIEW

File: manage.py [hunk 1]
```diff
@@ -25,14 +25,14 @@ def reassign_all_shipments():
         )
     )
     for shipment in shipments:
-        print "Making shimpent %s wait" % shipment['id']
+        print("Making shimpent %s wait" % shipment['id'])
```
Comment (sayatul) [inline manage.py]: change this to format

**Comment by sayatul** [comment]
update runtime file, docker file and, scripts folder also

File: manage.py (hunk 1 above)
Comment (sayatul) [inline manage.py]: change this to format

**Review by sayatul — CHANGES_REQUESTED** [review]

File: kitchensink/dropshipping.py [hunk 2]
```diff
@@ -70,12 +70,15 @@ def shipments():
 
 
 def match_quantum_data(shipments, quantum_data):
-    for shipment in shipments.values():
-        for row in quantum_data.values():
+    for shipment in list(shipments.values()):
```
Comment (sayatul) [inline kitchensink/dropshipping.py]: remove list

File: kitchensink/dropshipping.py [hunk 3]
```diff
@@ -70,12 +70,15 @@ def shipments():
...
+        for row in list(quantum_data.values()):
```
Comment (sayatul) [inline kitchensink/dropshipping.py]: same

File: kitchensink/dropshipping.py [hunk 4]
```diff
@@ -161,6 +164,7 @@ def get_open_drop_shipments():
             'shipment.delivery_address.full_address'
         ].split('\r\n'))
 
-    for shipment in shipments.values():
-        shipment['keywords'] = ' '.join(filter(None, set(shipment['keywords'])))
+    for shipment in list(shipments.values()):
```
Comment (sayatul) [inline kitchensink/dropshipping.py]: same

File: kitchensink/utils.py [hunk 5]
```diff
@@ -15,9 +13,11 @@ def client_url(model, id=None, domain=None):
     A filter for template engine to generate URLs that open
     on fulfil client of the instance::
         <a href="{{ 'product.product'|client_url }}">Product List</a>
-        <a href="{{ 'product.product'|client_url(product.id) }}">{{ product.name }}</a>
+        <a href="{{ 'product.product'
+        |client_url(product.id) }}">{{ product.name }}</a>
```
Comment (sayatul) [inline kitchensink/utils.py]: indent

File: kitchensink/utils.py [hunk 6]
```diff
@@ -15,9 +13,11 @@ def client_url(model, id=None, domain=None):
...
     A more sophisticated example with filter
-        <a href="{{ 'product.product'|client_url(domain=[('salable', '=', True)]) }}">
+        <a href="{{ 'product.product'
+        |client_url(domain=[('salable', '=', True)]) }}">
```
Comment (sayatul) [inline kitchensink/utils.py]: indent

**Review by sayatul — CHANGES_REQUESTED** [review]

**Comment by sayatul** [comment]
LGTM

File: kitchensink/dropshipping.py [hunk 7]
```diff
@@ -74,8 +74,11 @@ def match_quantum_data(shipments, quantum_data):
         for row in quantum_data.values():
             if (
                 (shipment['delivery_state'] is not None) and
-                (shipment['delivery_state'][-2:] == row['Ship To State/Province']) and
-                    (row['Ship To Postal Code'][-4:] == shipment['delivery_zip'][-4:])):
+                (shipment['delivery_state'][-2:]
```
Comment (tarunbhardwaj) [inline kitchensink/dropshipping.py]: not readable at all.

Flake8 should not override readability of code.

File: kitchensink/product.py [hunk 8]
```diff
@@ -14,7 +14,11 @@ def products():
         ('purchasable', '=', True),
         ('salable', '=', True),
     ], [('variant_name', 'ASC')],
-    ['rec_name', 'quantity_on_hand', 'quantity_available', 'quantity_inbound', 'quantity_outbound'])
+        [
```
Comment (tarunbhardwaj) [inline kitchensink/product.py]: why indent?

File: kitchensink/utils.py [hunk 9]
```diff
@@ -15,9 +13,11 @@ def client_url(model, id=None, domain=None):
     A filter for template engine to generate URLs that open
     on fulfil client of the instance::
         <a href="{{ 'product.product'|client_url }}">Product List</a>
-        <a href="{{ 'product.product'|client_url(product.id) }}">{{ product.name }}</a>
+        <a href="{{ 'product.product'
```
Comment (tarunbhardwaj) [inline kitchensink/utils.py]: not readable

File: manage.py [hunk 10]
```diff
@@ -58,25 +58,28 @@ def process_waiting_shipments():
...
 
     requests.post(
         os.environ['SLACK_WEBHOOK_URL'],
         json={
             'text': 'Tried to assign Mejuri Shipments',
             'attachments': [{
                 'title': 'Assigning Shipments',
-                'thumb_url': 'https://dto508s2j2p46.cloudfront.net/assets/mejuri_label-86c1982b0d84c5017186e435ff666485.png',
+                'thumb_url': (
+                    'https://dto508s2j2p46.cloudfront.net/assets/'
```
Comment (tarunbhardwaj) [inline manage.py]: http://example.com/path/to/thumb.png

File: manage.py [hunk 11]
```diff
@@ -58,25 +58,28 @@ def process_waiting_shipments():
             else:
                 break
         else:
-            print "Yuck! This one is pretty bad", shipment
+            print("Yuck! This one is pretty bad", shipment)
 
     requests.post(
         os.environ['SLACK_WEBHOOK_URL'],
         json={
             'text': 'Tried to assign Mejuri Shipments',
```
Comment (tarunbhardwaj) [inline manage.py]: `Tried to assign Shipments`

**Review by tarunbhardwaj — CHANGES_REQUESTED** [review]

File: kitchensink/utils.py [hunk 12]
```diff
@@ -15,9 +13,11 @@ def client_url(model, id=None, domain=None):
     A filter for template engine to generate URLs that open
     on fulfil client of the instance::
         <a href="{{ 'product.product'|client_url }}">Product List</a>
-        <a href="{{ 'product.product'|client_url(product.id) }}">{{ product.name }}</a>
+        <a href="{{ 'product.product'|client_url(product.id) }}">
+                 {{ product.name }}</a>
     A more sophisticated example with filter
-        <a href="{{ 'product.product'|client_url(domain=[('salable', '=', True)]) }}">
+        <a href="{{ 'product.product'
```
Comment (tarunbhardwaj) [inline kitchensink/utils.py]: Really man, I have to comment on this specifically don't you think it is quite obvious to apply same change everywhere?

File: manage.py [hunk 13]
```diff
@@ -58,25 +58,27 @@ def process_waiting_shipments():
...
 
     requests.post(
         os.environ['SLACK_WEBHOOK_URL'],
         json={
-            'text': 'Tried to assign Mejuri Shipments',
+            'text': 'Tried to assign Shipments',
             'attachments': [{
                 'title': 'Assigning Shipments',
-                'thumb_url': 'https://dto508s2j2p46.cloudfront.net/assets/mejuri_label-86c1982b0d84c5017186e435ff666485.png',
+                'thumb_url': (
```
Comment (tarunbhardwaj) [inline manage.py]: Ok, so why you need `(` now?

**Review by tarunbhardwaj — CHANGES_REQUESTED** [review]

**Review by tarunbhardwaj — APPROVED** [review]

**Pull request merged**